charts/cbcontainers-agent/cbcontainers-agent-chart/example-values.yaml

```diff
@@ -147,7 +147,7 @@ components:
   # if omitted, the cluster scanning components won't be installed
   clusterScanning:
     # enabled controls whether the cluster scanning features will be enabled
-    enabled: false
```

Comment:
Consider unifying this and the runtime value in the example - either both enabled (matches our setup-wizard flow) or both disabled (easier to understand for newcomers).

I don't mind either option, but having an example that only enables one of them is weird to me and would probably be confusing.